File: android/app/src/main/java/com/phicdy/totoanticipation/view/activity/SettingActivity.java
package com.phicdy.totoanticipation.view.activity;

import android.os.Bundle;
import android.preference.PreferenceActivity;

import com.phicdy.totoanticipation.view.fragment.SettingFragment;


public class SettingActivity extends PreferenceActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
    	super.onCreate(savedInstanceState);
		final SettingFragment fragment = new SettingFragment();
		getFragmentManager().beginTransaction()
				.replace(android.R.id.content, fragment)
				.commit();
	}
}

File: android/app/src/main/java/com/phicdy/totoanticipation/view/activity/SettingActivity.kt
package com.phicdy.totoanticipation.view.activity

import android.os.Bundle
import android.preference.PreferenceActivity

import com.phicdy.totoanticipation.view.fragment.SettingFragment


class SettingActivity : PreferenceActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val fragment = SettingFragment()
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, fragment)
                .commit()
    }
}
